apps/dotcms-ui/src/app/portlets/dot-edit-page/content/dot-edit-content.component.html

```diff
@@ -1,9 +1,11 @@
+<dot-content-palette (filterChange)='loadContentPallet($event)' [items]='contentPalletItems' *ngIf="(pageState$ | async).state.mode === dotPageMode.EDIT"></dot-content-palette>
 <dot-form-selector
     [show]="editForm"
     (select)="onFormSelected($event)"
     (close)="editForm = false"
 ></dot-form-selector>
 <dot-add-contentlet (custom)="onCustomEvent($event)"></dot-add-contentlet>
```

Comment:
Not now, but there is a way to load this dynamically and I think we should do it.